### DISCUSSION TITLE
Problem:  {"error":"Inventory ID in body and URL must be the same"}

### DISCUSSION THREAD

**Discussion by ghost** [semaphoreui/semaphore#2346] [Q&A] [answered]
### Issue

Greetings! I really like your product, I want to automate the launch of the anisble playbook, but unfortunately, when I try to update the inventory using the PUT method along this path /api/project/2/inventory/14, it fails. Gives an error {"error":"Inventory ID in body and URL must be the same"}
My request:
```
curl -v -X 'PUT' \
-H 'Accept: application/json' \
-H 'Content-Type: application/json' \
-H 'Authorization: Bearer example' \
-d '{
  "inventory": "test"
}' \
https://example/api/project/2/inventory/14
```

On this path there is a valid inventory with the same id that I specify in the url
### Impact

Web-Backend (APIs)

### Installation method

Docker

### Database

MySQL

### Semaphore Version

v2.10.22-e44910d-1721658561

**Comment by fiftin** [comment, marked as answer]
Hi,
you should provide `"id": 14` in the request body. It is old issue.
